bpf/alignchecker/bpf_alignchecker.c

```diff
@@ -3,6 +3,8 @@
 #include "include/vmlinux.h"
 #include "include/api.h"
 #include "lib/hubble_msg.h"
+#include "process/retprobe_map.h"
+#include "process/types/basic.h"
```

Comment:
I believe that this include statement is not required here.

Reply:
If not include them, the checker will output 

```
C and Go structs alignment check failed: tracingapi.EventConfig(80) size does not match event_config(-1)
```

I add a print before the 

https://github.com/cilium/tetragon/blob/77e32bf8f35931c84dda6f5eef1be9bb04a15ec0/vendor/github.com/cilium/cilium/pkg/alignchecker/alignchecker.go#L95

```
fmt.Printf("%v, %v\n", st.StructName, st.ByteSize)
```

And it turns out that the struct size is not read correctly.

```
event_config, -1
msg_exit, 40
msg_test, 48
```